### DISCUSSION TITLE
No way to change master password on Mac

### DISCUSSION THREAD

**Discussion by Pyanne** [keepassxreboot/keepassxc#9046] [Q&A] [answered]
Hi everybody!

I cannot change the database master password on Mac.
The user guide says this feature is in Settinfgs>Security>Database credentials. I do not see this tab.
Where is it ?
![missingtab](https://user-images.githubusercontent.com/11357449/215701254-c790b0fe-ef9d-4cfb-bef8-cb046c7d25aa.png)

**Comment by michaelk83** [comment, marked as answer]
It looks to me that you're looking in Application Settings instead of Database Settings.
https://keepassxc.org/docs/KeePassXC_UserGuide.html#_database_settings

**Comment by Pyanne** [comment]
You are right!
I should have looked in Database>Security instead of using the setting button:  ![Screenshot 2023-01-31 at 10 43 44](https://user-images.githubusercontent.com/11357449/215724894-4f0c44cb-4b4d-441c-a607-9f981508506d.png)
Thanks.
